msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.CSharp.props

```diff
@@ -19,25 +19,6 @@ Copyright (C) 2014 Xamarin Inc. All rights reserved.
 	<Import Project="$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props"
 		Condition="Exists('$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props')"/>
 
-	<PropertyGroup Condition="'$(OS)' == 'Unix'">
```

Comment:
Some bits moved to Xamarin.Mac.ObjCBinding.Common.props as fits better and easier to depend on Xamarin.Mac.Common.props import.